lib/article_json/export/apple_news/elements/paragraph.rb

```diff
@@ -0,0 +1,32 @@
+module ArticleJSON
+  module Export
+    module AppleNews
+      module Elements
+        class Paragraph < Base
+          # Generate the paragraph node with its containing text elements
+          # @return [Hash]
+          def export
+            {
+              role: 'body',
+              text: text
```

Comment:
I considered that, as we are returning HTML, that we should possibly tag paragraphs in 'p' tags  in order to maintain consistent HTML, but also thought it may be superfluous. 


Opinions?

```suggestion
              text: "<p>#{text}</p>"
```

Reply:
In my opinion, it makes sense, because of the documentation comment 

> `# Generate the paragraph node with its containing text elements`

Reply:
This depends on how Apple defines the format. From the example [here](https://developer.apple.com/documentation/apple_news/body#2973276) I understand that it's not necessary and that `{"role":  "body","text": "some text of the paragraph"}` is a valid paragraph.

Paragraph is what we call this type of element, Apple calls it body. So I don't think it needs to include a `p` tag...